src/config/ch.sbb.regionenkarte.public/index.js

```diff
@@ -65,7 +65,6 @@ export const regionenkartePublicSegment = new Layer({
       },
       properties: {
         hideInLegend: true,
-        showPopupOnHover: true,
```

Comment:
@oterral We don't need this here anymore right? It was preventing the select highlight from lingering

Reply:
And do we even need the whole showPopupOnHover functionality? I don't think we use it anywhere

Reply:
If the `showPopupOnHover` property is not used anywhere, we can remove the functionnality, I guess. But check on the gitlab project too  before you do it.